.github/ISSUE_TEMPLATE/Documentation.yml

```diff
@@ -0,0 +1,25 @@
+name: Documentation
+description: Report an issue related to the SciPy documentation.
+title: "DOC: "
+labels: [Documentation]
+
+body:
+- type: textarea
+  attributes: 
+    label: "Issue with current documentation:"
+    description: >
+      Please make sure to leave a reference to the document/code you're
+      referring to. You can also check the development version of the
+      documentation and see if this issue has already been addressed at
+      https://https://scipy.github.io/devdocs/.
+
+- type: textarea
+  attributes:
+    label: "Idea or request for content:"
+    description: >
+      Please describe as clearly as possible what topics you think are missing
+      from the current documentation.
```

Comment:
Maybe invite suggestions for phrasing? Not that PR authors would be bound to it, but I think that might help us understand what the user is looking for.

Reply:
How about this?